spec/service_spec.rb

```diff
@@ -109,38 +109,55 @@
   end
 
   context "#get_resources_by_uri_path" do
-    it "fetches data_center" do
-      return_message = <<-EOX.chomp
+    let(:return_message) do
+      <<-EOX.chomp
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <data_center href="/api/datacenters/00000001-0001-0001-0001-0000000000f1" id="00000001-0001-0001-0001-0000000000f1">
-    <name>Default</name>
-    <description>The default Data Center</description>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/storagedomains" rel="storagedomains"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/clusters" rel="clusters"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/networks" rel="networks"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/permissions" rel="permissions"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/quotas" rel="quotas"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/qoss" rel="qoss"/>
-    <link href="/api/datacenters/00000001-0001-0001-0001-0000000000f1/iscsibonds" rel="iscsibonds"/>
-    <local>false</local>
-    <storage_format>v3</storage_format>
+  <name>Default</name>
```

Comment:
The 2 spaces is fine, I didn't know if there was another reason that it changed in this PR.